### PR TITLE
Create 21-11-11-GitLab-Unauthenticated-RCE-Flaw

### PR DESCRIPTION
**Pull request by MrIamGr00t**

### CODE REVIEW

**Review by cookiemonster — APPROVED** [review]

File: csirt.divd.nl/_posts/21-11-11-GitLab-Unauthenticated-RCE-Flaw [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (MrSeccubus) [inline csirt.divd.nl/_posts/21-11-11-GitLab-Unauthenticated-RCE-Flaw:1]: This file is missing the .md extentsion

**Review by MrSeccubus — CHANGES_REQUESTED** [review]
Give the file a .md extension pelase!

**Comment by MrSeccubus** [comment]
Don;t need this PR

**Comment by MrSeccubus** [comment]
Closed the wrong one.

**Pull request closed without merging**